### ISSUE TITLE
t/02_timezone.t fails (with recent DateTime::TimeZone?)

### ISSUE THREAD

**Issue by eserte** [karupanerura/Time-Strptime#12]
Log output:
```
            #   Failed test at t/02_timezone.t line 52.
            #          got: '-654768001 36000'
            #     expected: '-654764401 32400'
            # Looks like you failed 1 test of 1.

        #   Failed test 'detect time_zone from option'
        #   at t/02_timezone.t line 53.

            #   Failed test at t/02_timezone.t line 57.
            #          got: '-654768001 36000'
            #     expected: '-654764401 32400'
            # Looks like you failed 1 test of 1.

        #   Failed test 'detect time_zone from env'
        #   at t/02_timezone.t line 58.

            #   Failed test at t/02_timezone.t line 63.
            #          got: '-654768001 36000'
            #     expected: '-654764401 32400'

            #   Failed test at t/02_timezone.t line 64.
            #          got: '-654768001 36000'
            #     expected: '-654764401 32400'
            # Looks like you failed 2 tests of 2.

        #   Failed test 'detect time_zone from date time string'
        #   at t/02_timezone.t line 65.
        # Looks like you failed 3 tests of 3.

    #   Failed test '1949-04-03 01:59:59'
    #   at t/02_timezone.t line 66.
... (etc) ...
```

Statistical analysis identifies DateTime::TimeZone 2.17 as the version breaking the tests:
```
****************************************************************
Regression 'mod:DateTime::TimeZone'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      1.0000	      0.0000	12628586807524504.00
[1='eq_2.01']  	     -0.0000	      0.0000	  -0.61
[2='eq_2.09']  	     -0.0000	      0.0000	  -1.02
[3='eq_2.11']  	     -0.0000	      0.0000	  -1.49
[4='eq_2.13']  	     -0.0000	      0.0000	  -0.69
[5='eq_2.14']  	     -0.0000	      0.0000	  -1.34
[6='eq_2.15']  	     -0.0000	      0.0000	  -1.38
[7='eq_2.16']  	      0.0000	      0.0000	   0.00
[8='eq_2.17']  	     -1.0000	      0.0000	-12308823847850894.00

R^2= 1.000, N= 113, K= 9
****************************************************************
```

**Comment by karupanerura** [comment]
Thank you for reporting.
Fixed it by 554abea temporary.

**Comment by karupanerura** [comment]
Sorry for later. This problem is fixed on 1.03. Please check it. Thank you for reporting!
